rtloader/three/three.cpp

```diff
@@ -159,7 +164,21 @@ py_info_t *Three::getPyInfo()
     Py_XDECREF(str_path);
     return info;
 }
+/**
+ * freePyInfo()
+ */
 
+void Three::freePyInfo(py_info_t* info) {
+    if(info->version){
```

Comment:
Note: It is not needed to check for null for `free`: https://stackoverflow.com/questions/6084218/is-it-good-practice-to-free-a-null-pointer-in-c